proxy/deploy.go

```diff
@@ -25,7 +25,9 @@ func DeployFunction(fprocess string, gateway string, functionName string, image
 		fprocessTemplate = "ruby index.rb"
 	} else if language == "csharp" {
 		fprocessTemplate = "dotnet ./bin/Debug/netcoreapp2.0/root.dll"
-	}
+	} else if language == "php" {
+    fprocessTemplate = "php index.php"
```

Comment:
Make sure to run `go fmt`

Reply:
@kylestev do you want to add `gofmt` / a liniting check to the CI script?